src/routes/InputCardPage.tsx

```diff
@@ -91,8 +91,8 @@ export default function InputCardPage() {
           BundleOption.decode,
           O.fromEither,
           O.chain((r) => O.fromNullable(r.belowThreshold)),
-          O.fold(
-            () => undefined,
+          O.foldW(
+            () => onError,
```

Comment:
Please note that you are defining an anonymous function that when executed will not call the onError function